js/entity_browser.widgets.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @file entity_browser.widgets.js
+ *
+ * Defines the behavior of the entity browser's config widgets page.
+ */
+(function ($, Drupal, drupalSettings) {
+
+  "use strict";
+
+  /**
+   * Registers behaviours related to config widgets page.
+   */
+  Drupal.behaviors.entityBrowserWidgets = {
+    attach: function () {
+      $('#edit-widget').on('change', function(e) {
```

Comment:
This is not needed and is actually messing with AJAX if it is done correctly. One of the steps that I needed to do was to remove this entirely.